src/components/meta.tsx

```diff
@@ -14,6 +14,12 @@ const Meta = () => {
       <meta name="theme-color" content="#fefcfa" />
 
       <link rel="alternate" type="application/rss+xml" href="/feed/rss.xml" />
+
+      <script src="https://getinsights.io/js/insights.js"></script>
+      <script>
+        insights.init('nKQvSs_BBESe7Jbn');
```

Comment:
We should only enable the analytics on prod:
 
```suggestion
        insights.init('nKQvSs_BBESe7Jbn', { disabled: process.node_env !== "production"});
```

(not sure about the proposed condition, we have to test it)